src/components/Chat/SendMessageButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { IconSend } from '@tabler/icons-react';
+
+import classNames from 'classnames';
+
+import { ConversationsSelectors } from '@/src/store/conversations/conversations.reducers';
+import { useAppSelector } from '@/src/store/hooks';
+import { ModelsSelectors } from '@/src/store/models/models.reducers';
+
+interface Props {
+  messageIsStreaming: boolean;
```

Comment:
I think you can use just selector inside component

Reply:
Good suggestion.
I've changed to use selector instead.